src/factories/routePathLinkFactory.ts

```diff
@@ -10,7 +10,11 @@ class RoutePathLinkFactory {
     public static createRoutePathLink = (routePathLinkNode: any): IRoutePathLinkResult => {
         const nodes = [];
         if (routePathLinkNode.solmuByLnkalkusolmu) {
-            nodes.push(NodeFactory.createNode(routePathLinkNode.solmuByLnkalkusolmu));
+            const node = NodeFactory.createNode(routePathLinkNode.solmuByLnkalkusolmu);
+            if (routePathLinkNode.relpysakki === 'E') {
+                node.disabled = true;
+            }
+            nodes.push(node);
```

Comment:
Correct me if I'm wrong, but won't this cause problems when two routes use the same stop? What if the stop is disabled on one of the routes, but not the other. This code may mark one of the stops as disabled, but leave the other stop from the other routes active. If looking at function that calls this code, we can see that it collect all nodes, and remove duplicates by looking at ID. The duplicate function wont recognise difference in activity. This leads to a problem that where we have a race condition that sets the outcome. If the active stop is before the disabled stop in the list, it will lead the merged stop to be active. However, if the disabled stop is before, it will lead the merged stop to be disabled.